core/src/Kokkos_HBWSpace.hpp

```diff
@@ -100,26 +100,13 @@ class HBWSpace {
   using memory_space = HBWSpace;
   using size_type    = size_t;
 
-  /// \typedef execution_space
+  /// \execution_space alias
```

Comment:
Is this change intended?
```suggestion
  /// \typedef execution_space
```